build/images/wireguard-go/README.md

```diff
@@ -0,0 +1,17 @@
+# images/wireguard-go
+
+This Docker image is a very lightweight image based on Ubuntu 20.04 which
+includes WireGuard golang implementation and wireguard-tools. It can be used
+for Kind clusters for tests when injected as a sidecar to antrea-agent.
+
+If you need to build a new version of the image and push it to Dockerhub, you
+can run the following:
+
+```bash
+cd build/images/wireguard-go
+docker build -t antrea/wireguard-go:latest .
```

Comment:
Please update the command accordingly, refer to other README.

Reply:
ok.